### ISSUE TITLE
False positive errors when trying to add multiple rules

### ISSUE THREAD

**Issue by dastergon** [thombashi/tcconfig#84]
Hi, 

When I try to add multiple incoming/outgoing rules to distinct IPs I get an error from tcconfig. However, the error seems to be false positive since when I use `tcshow`, it shows that the rules have been applied.

### How to reproduce

List the rules for the network iface:

```
# tcshow -d ens3
{
    "ens3": {
        "outgoing": {},
        "incoming": {}
    }
}
```

Set outgoing/incoming rules for a specific dst-network:

```
# tcset --direction outgoing --dst-network 192.168.122.205 --rate 500Mbps --device ens3 --add; tcset --direction incoming --dst-network 192.168.122.205 --device ens3 --rate 500Mbps --add
```
List available rules:
```# tcshow -d ens3
{
    "ens3": {
        "outgoing": {
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M"
            }
        },
        "incoming": {
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M"
            }
        }
    }
}
```

Set new incoming/outgoing rules to another IP:
```
# tcset --direction outgoing --dst-network 192.168.122.201 --rate 500Mbps --device ens3 --add; tcset --direction incoming --dst-network 192.168.122.201 --device ens3 --rate 500Mbps --add
[NOTICE] tcconfig: failed to add ip link: ip link already exists. execute with --overwrite option if you want to overwrite the existing rules. execute with --add option if you want to add a new rule in addition to the existing rules.
[NOTICE] tcconfig: failed to 'tc qdisc add': ingress qdisc already exists. execute with --overwrite option if you want to overwrite the existing rules. execute with --add option if you want to add a new rule in addition to the existing rules.
```
However, if I list the available rules in the interface, it shows that the new rules have been applied successfully:

```
# tcshow -d ens3
{
    "ens3": {
        "outgoing": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "500M"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M"
            }
        },
        "incoming": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "500M"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M"
            }
        }
    }
}
```

Thanks in advance.

**Comment by thombashi** [comment]
@dastergon 
Thank you for your feedback.
I've fixed to suppress the message at `tcconfig 0.14.0`.
(the message also appears when I use `--change`)

```
+ tcset --direction outgoing --dst-network 192.168.122.205 --rate 500Mbps --device ens33 --overwrite
+ tcset --direction incoming --dst-network 192.168.122.205 --device ens33 --rate 500Mbps --add
+ tcshow -d ens33
{
    "ens33": {
        "outgoing": {
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        },
        "incoming": {
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        }
    }
}
+ tcset --direction outgoing --dst-network 192.168.122.201 --rate 500Mbps --device ens33 --add
+ tcset --direction incoming --dst-network 192.168.122.201 --device ens33 --rate 500Mbps --add
+ tcshow -d ens33
{
    "ens33": {
        "outgoing": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::801"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        },
        "incoming": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::801"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        }
    }
}
+ tcset --direction outgoing --dst-network 192.168.122.201 --rate 300Mbps --device ens33 --change
+ tcset --direction incoming --dst-network 192.168.122.201 --device ens33 --rate 300Mbps --change
+ tcshow -d ens33
{
    "ens33": {
        "outgoing": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "300M",
                "filter_id": "800::801"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        },
        "incoming": {
            "dst-network=192.168.122.201/32, protocol=ip": {
                "rate": "300M",
                "filter_id": "800::801"
            },
            "dst-network=192.168.122.205/32, protocol=ip": {
                "rate": "500M",
                "filter_id": "800::800"
            }
        }
    }
}
```

Note: `"filter_id"` entry added to `tcshow` output to support #80.

**Comment by dastergon** [comment]
@thombashi  thank you very much for the quick fix!
